src/main/java/org/myrobotlab/service/abstracts/AbstractSpeechSynthesis.java

```diff
@@ -200,13 +203,14 @@ public Object getVoiceProvider() {
 
   public AbstractSpeechSynthesis(String reservedKey) {
     super(reservedKey);
-    
+    setReady(false);
     audioFile = (AudioFile) createPeer("audioFile");
-    
+
```

Comment:
isReady = false;

by doing this the default for all SpeechSynthesis is false ...  maybe its a good default, I guess out of all of them Mary, Local are nearly the only ones self contained

Reply:
we can add some check later to localspeech / mimic, 
mary need some time to load and ready status is set only when loaded.